site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -295,6 +295,17 @@ the steps in [Viewing Warnings](#view_warnings).
       <td>Grants the extension access to the <a href="/docs/extensions/reference/proxy">chrome.proxy</a> API.</td>
       <td><strong>Read and change all your data on all websites</strong></td>
     </tr>
+    <tr id="sessionshistory">
+      <td><code>"sessions"</code> and <code>"history"</code></td>
+      <td>Grants the extension access to the <a href="/docs/extensions/reference/sessions">chrome.sessions</a> API and <a href="/docs/extensions/reference/history">chrome.history</a> API.</td>
+      <td><strong>Read and change your browsing history on all your signed-in devices</strong></td>
+    </tr>
+    <tr id="sessionstabs">
+      <td><code>"sessions"</code> and <code>"tabs"</code></td>
+      <td>Grants the extension access to the <a href="/docs/extensions/reference/sessions">chrome.sessions</a> API and privileged fields of the <a
```

Comment:
Can you link to the privileged fields or do we need to document them?

Reply:
We describe the privileged fields in the [Permissions](https://developer.chrome.com/docs/extensions/reference/tabs/#perms) section of the Tabs API

Reply:
Please add a link.

Reply:
Done 👍